pe-terminal/deb/debian/rules

```diff
@@ -11,3 +11,9 @@ override_dh_auto_build:
 
 override_dh_auto_clean:
 	-rm -rf go-workspace
+
+override_dh_installinit:
+	dh_installinit --no-scripts
+
+override_dh_systemd_enable:
```

Comment:
Why was it necessary to add this section?  Can you copy the build error here?  My understanding of this override is that it's only needed if the package name differs from the service and/or service file name, which doesn't seem to be the case here since all these items are called pe-terminal, so for my education I'd like to understand what happened.

Reply:
Build was throwing below error prior to adding this.

```bash
pe-terminal: init.d-script-not-included-in-package etc/init.d/pe-terminal

The /etc/init.d script is registered in the postinst script, but is not included in the package.
Severity: error
Check: init.d
```